Index: backend_api/obj/rider.project.restore.info
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>16225629111735328
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/backend_api/obj/rider.project.restore.info b/backend_api/obj/rider.project.restore.info
--- a/backend_api/obj/rider.project.restore.info	(revision e1e76ed76a9bcd7425b6346f1344be26f83b42a7)
+++ b/backend_api/obj/rider.project.restore.info	(date 1622644931267)
@@ -1,1 +1,1 @@
-16225629111735328
\ No newline at end of file
+16226449312438978
\ No newline at end of file
